README.md

```diff
@@ -19,10 +19,13 @@ The game client is very slow in debug mode, so it should be run in release mode
 SDL2 needs to be added to be able to compile Plen under Windows. You can follow this tutorial https://github.com/Rust-SDL2/rust-sdl2/blob/master/README.md#windows-with-build-script
```

Comment:
While you're at it :)

`you can follow [this tutorial](<url>)` for better readability

Reply:
Fixed